yarn.lock

```diff
@@ -1112,9 +1112,9 @@ ajv@^6.10.0, ajv@^6.12.3, ajv@^6.12.4:
     uri-js "^4.2.2"
 
 ajv@^8.8.2:
-  version "8.8.2"
-  resolved "https://registry.yarnpkg.com/ajv/-/ajv-8.8.2.tgz#01b4fef2007a28bf75f0b7fc009f62679de4abbb"
-  integrity sha512-x9VuX+R/jcFj1DHo/fCp99esgGDWiHENrKxaCENuCxpoMCmAt/COCGVDwA7kleEpEzJjDnvh3yGoOuLu0Dtllw==
+  version "8.10.0"
+  resolved "https://nexus-proxy-prd.soma.salesforce.com/nexus/content/groups/npm-all/ajv/-/ajv-8.10.0.tgz#e573f719bd3af069017e3b66538ab968d040e54d"
```

Comment:
It's resolving against `https://nexus-proxy-prd.soma.salesforce.com/` which is wrong. It should be resolving against `https://registry.yarnpkg.com/` in this repo.

Reply:
@maliroteh-sf so how did this happen?  I didn't specify to use soma.salesforce.com.  🤷

Reply:
My guess is that you have .npmrc pointing to nexus and you did yarn install.